### ISSUE TITLE
`Lazy-loading` Bug report: Application crashed with AttributeError

### ISSUE THREAD

**Issue by superstar54** [aiidalab/aiidalab-qe#920]
## Automated report

_This issue was created with the app's automated bug reporting feature.
Attached to this issue is the full traceback as well as an environment
fingerprint that contains information about the operating system as well as all
installed libraries._

## Additional comments (optional):

_Example: I submitted a band structure calculation for Silica._

## Attachments

<details>
<summary>Traceback</summary>

```python-traceback
/opt/conda/lib/python3.9/site-packages/ipywidgets/widgets/widget.py in notify_change(self, change)
    692                 # Send new state to front-end
    693                 self.send_state(key=name)
--> 694         super(Widget, self).notify_change(change)
    695 
    696     def __repr__(self):

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in notify_change(self, change)
   1515     def notify_change(self, change):
   1516         """Notify observers of a change event"""
-> 1517         return self._notify_observers(change)
   1518 
   1519     def _notify_observers(self, event):

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in _notify_observers(self, event)
   1562                 c = getattr(self, c.name)
   1563 
-> 1564             c(event)
   1565 
   1566     def _add_notifiers(self, handler, name, type):

/opt/conda/lib/python3.9/site-packages/traitlets/traitlets.py in _update_source(self, change)
    377             return
    378         with self._busy_updating():
--> 379             setattr(self.source[0], self.source[1], self._transform_inv(change.new))
    380             if getattr(self.target[0], self.target[1]) != change.new:
    381                 raise TraitError(

~/apps/aiidalab-qe/src/aiidalab_qe/plugins/xas/setting.py in <lambda>(value, kind_name)
    165                     lambda kinds, kind_name=kind_name: kinds.get(kind_name, False),
    166                     lambda value, kind_name=kind_name: {
--> 167                         **self._model.elements,
    168                         kind_name: value,
    169                     },

AttributeError: 'input_structure' object has no attribute 'elements'
```
</details>

<details>
<summary>Environment fingerprint</summary>
<pre>eNptWF1v6zYS_SuB97UyrA87yX27m-0CWzTdbHO7RbFYGJRES4wpkhEpO0px_3uHM0Nb6e2jD0lxOHPmzIx_X53k6JU1q0_5dyun
RTjYcVh9-n0lxqZXQTZhGuXq0_9Wu6pWYfXdavV_2DeH3pr95eiqXN-v8xJWr9Dfqtvsl3oyYbp5fny6efr5--8fn77s__HbT58f
__Vw889R3TxLd1Pc3hT5p2rzqdjd_PLl4abYFNXqK1whmqPopEdTmkZ6r2otMzd3gzQB4NVmvVlXcKVQqhVZY6OZq2K9WxcX0IGV
1s2A5-t8sdl5ObUW4d16c4FfJwHmDtK7Ee6L69WH9ZOqR-ub6xfzy9JZGKPkeL9BG4rFoctKdrbj8aDt2eOe8rpHizpCFThxie17
O8i_XHhlOAcfiHK5clZtJ4Pf18LLdI0gh9jMqaMAcLe-Y8sh1K8YvZIA-ILwQY7o3VuKqNByqFWDL-
ZzxtgggmyzMDvpeTMaaLwq-jBo3H1PkTCzsnjJjk_PZCSeY9oI51o1ejxWUaCcG2wrcdMdfTwS8gQUEKFHuKLvj501RdYcDio-
OE-OvcJZrUyrTOdpnYOD_inXRUFWeZM34-wCsWLLYAj2KA0FDOxAbDbNvpNGjiLYkYiAHwwBHxA_iDf8XdRS48mc7qiB0o3QGo0n
KyLk7Bj8-jCZJlirfabHKWtE00vmZ_RGjbYhIfmgFFNQh0l7O7kKF-CW6A_IEjgMyBYsxmu1MkdJlm7jb3hSj3ZV6-h74HTQCjmd
4lhPgzvhEUhH4EZe3AP6EI16sCaMlt6QE23QWLSdDawQTP6ATWhYI0ewWNFHK7ipSiDfTV7iQIJXt_y7O2GkKjSt6cXoZcgMcFdo
9Y5WluzyRo3NRITMiTSNVs0RAGDu-
jb9zrxTxjDNOXyN1XYUg2Bm7RAyraCIUv4hkLEyZb0wLbi2Y67zhmCnkfWhZAw8CfsBuuVoEtG6Ubg-7izvWMmaudHJKnJGK-
up48_tMFqtBKVj5m1Zglp5mLxs3wadEhdBUDGIAqlxImWr_HFv6xfQdR9IMvPrEvCgxk-AnkfEeFJ6_ABRo7WNDyM8e-9iINjaHa
0AJTW7n42Y9Fk1lK6caRL0BBScszgJrByE0vsTRLTlxyUVhb3gLquS6EdHySkqO4tSieySb4108bHdCBmBCwUvyGgXx4kcdoDMfv
HWeCAuhhxMo7pxUFpqi5QpI4a7tQp7ri8le_cAkb5SflsROFGxhHs2aDuoXMZqCZaxZ_AeDP5JvZNnNhjabpTSaBTGaGiE-jynQ0
T-fotcKBM9-mkQ5jAqCUxEktC1qgWnqcPM9TBHyAgyP_4YYmWF3GG9xQ1D1CGIfzbIICAIqVJsPixCYQSCN6j623W6D0JhzUF1rJ
OIuRlEx6AC7qInGbQQpEGwsBcJZZLdrstqCWWgs1dOXXeDoxgpCOGyh58gYbmCmXwL4CDZLgiggm-F6SRwxS-49iJbldQjbvtBmR
eoniQw8asvtqYUybn6RBZtuZJt-feFVVV698sERstxD-oDkaGH0va0QOyCA2Qf4xkk2ImlO7__sBIwjyA1mFSLa_qpRpM3S5B6jK
iidx_Ajx0VOfS6ur8YUEQ9_tPi1et5ktEjtAsKW6uk50d1vhaXWwa81eldFYJajMdF0kIfCgkN9b9ZvETLN9VS-
Yi3baLoQDQGMdSCNZJKhrbWy2snmpT4URwt3xFzC88tTj2CCa09m0uaRGByz-IgWUDpVHDaslCW3ONcwUwZqAmSgxItHEBWJyM5N
SKJBtCOOZVBSKEi3j64gRQxWTv4LlaahUAPkw6qVU2glELM1I0W0Bs3XLe2jDHPUkEHyBrwCHGPAmPq2OqLgJlMKmGkB8GKHY5Kf
dAO4RB71zdqmGhnp09Knjk6kVIGoiXNiash7rFB1tYe0dzYnyywzPdqWFDXQMdBISyoRlrob_rkZbpzIR1RYKNlVItJ3AvqRhwwM
AQBbRB3tiWCg9RiocwAvLpFH-6gmIskBSUDtoFyAFz37AvaCF2CQiJhY8eYt9yp4ln55mRD7RoJqOvuNpsNRZcKgOv8xFlS4c8Av
ufw4zMuowtYxFdD66KlhwZIcmi3iGqYKeLJ9LYnxbEv0BjlaFjAW49d1NN9lHHIwL0Rg0yX4hU8AHIzDg0iuTmSe0aiFFQunJ66b
l76RXMEI58xfZzmWoX7Rwkj2jCogBh7ZYQBJ_Ry8tmCsTnfAIsuZLHIHvkUlElc8fENaA2ln_MzDGVdgc66x9YlQbH3F-P8caUzg
htYUlsHc8xo44y5SBkHpTyTJ6EXwujmGnSJACpUT_ODOvz8K_qKukYHF3tBT6GWDZBLr1RQtgM0jRqTsSL1cTMUpIB5XHD-JGh_m
W0LZjMvQHkIIQ02nMJxLQjocHtl3jJw7iBTr16ivRe1L1KH_TT_8OsXBDiUM6gCVN4kUTvixtP8OD__50cudvSxn8SDXpJg_reT5
vn5x-Uc5Ob4esrRGEMMwgwITJop5HdsyDO0XpRtt_waTotbHgsJACqdeMLZXMH4cCDuB3twYZRdHOi5Bbs-E1sMOCWZTwXwslisg
FaxoG2SNRGPNT7Ttus4opfFUczcPbOp76lakbd--_z4I4rhhq95x_E7Ugk_8Sqz1FOmjmSUrxM8gf8y-IgtpkQSyHGKMreeBQ8C0
IQXf8LXTWrxC2QrLWVxKUvzziYS4y7e5BuV_usgVfINqJ_juh3Vx0t5dKlw3WPeezmceCqjdz5D91V8GYXv-U8E_LQMk0uv3W25C
_U99JnTpNrk9eg4r97IBmqmvFEwItpFqfTGnmsYrIFQwyCXw0Ocjr2CXoDnXbzDdeSDgkfKZ8wVauTpFCUPVCBbT0q3i1pdXpb3L
bTQneG6m1_wOAOO0AgI57TspXb8lOqbHa08LdaLb9bjHym8IZHs44YXf-kZ_mr5NSw-X36zDIKk4gQNmXn5y4b-IYAs_6xjEzvz1
SUaH6DYZjwZbLhpAnCM9ZZEpqLa5ufhSpt4MZTjKbZ0rB3Rx5BaolHYB91zGkOlHUCtuXvkaId-
lKJ1QJQmaCQVxQh0b268LxZNfbCJ9-TN-FstnBfZ9p6ma_yGHfk6Gv7CazvQTEoBhYRWQVOHu012zy4OvzhTJEVJVT68s3eQLfGL
U9SK5ag0mfiPh4AKrXCGKrmyTO6wD3bP29M0ATXi-kcHZ11JOFC4ZKLj-ROMYCOdZZn4r4ChbV7MnCcFuQXdEWoazJwFaexZhKZv
bYcP2VyhOAr7Za07N9Dw839vBdp3lsLvF__m5dTknWUtDXTvXJoSVwD2oO8wE1-qfZo-L0so_qQ1515KimfJhuJFewe8tbENvvR4
-WLZm_oSG_7XEU2CeTssVeRt0IvpnwycxchpQF98V87RBbEgf_36B8BHl3U=</pre>
</details>

**By submitting this issue I confirm that I am aware that this information can
potentially be used to determine what kind of calculation was performed at the
time of error.**

**Comment by superstar54** [comment]
I run an `XAS` calculation with `Si` bulk. After submitting the job, I click the `job-history` button, and get this error.

**Comment by edan-bainglass** [comment]
Thanks @superstar54. This is resolved in the merged version of #802.
